src/expr/src/relation/func.rs

```diff
@@ -2094,7 +2098,9 @@ impl RustType<ProtoCaptureGroupDesc> for CaptureGroupDesc {
     }
 }
 
-#[derive(Arbitrary, Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash, MzReflect)]
```

Comment:
Yes, good idea!

Reply:
I just checked: We can't remove the `Hash` impls yet because there are other crates that I didn't get to yet that still put `Mir*Expr`s into `HashMap`s. So this needs to be a follow-up for other these crates have been moved away from `HashMap`s too.